pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -178,43 +175,44 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Add a connector for a specific domain.
-		#[pallet::weight(T::WeightInfo::add_connector())]
+		/// Add a known instance of a deployed liquidity pools integration for a
+		/// specific domain.
+		#[pallet::weight(T::WeightInfo::add_instance())]
 		#[pallet::call_index(1)]
-		pub fn add_connector(origin: OriginFor<T>, connector: DomainAddress) -> DispatchResult {
+		pub fn add_instance(origin: OriginFor<T>, instance: DomainAddress) -> DispatchResult {
 			T::AdminOrigin::ensure_origin(origin)?;
 
 			ensure!(
-				connector.domain() != Domain::Centrifuge,
+				instance.domain() != Domain::Centrifuge,
 				Error::<T>::DomainNotSupported
 			);
 
 			ensure!(
-				!ConnectorsAllowlist::<T>::contains_key(connector.domain(), connector.clone()),
-				Error::<T>::ConnectorAlreadyAdded,
+				!Allowlist::<T>::contains_key(instance.domain(), instance.clone()),
+				Error::<T>::InstanceAlreadyAdded,
 			);
 
-			ConnectorsAllowlist::<T>::insert(connector.domain(), connector.clone(), ());
+			Allowlist::<T>::insert(instance.domain(), instance.clone(), ());
 
-			Self::deposit_event(Event::ConnectorAdded { connector });
+			Self::deposit_event(Event::InstanceAdded { instance });
 
 			Ok(())
 		}
 
-		/// Remove a connector from a specific domain.
-		#[pallet::weight(T::WeightInfo::remove_connector())]
+		/// Remove a instance from a specific domain.
+		#[pallet::weight(T::WeightInfo::remove_instance())]
 		#[pallet::call_index(2)]
-		pub fn remove_connector(origin: OriginFor<T>, connector: DomainAddress) -> DispatchResult {
+		pub fn remove_instance(origin: OriginFor<T>, instance: DomainAddress) -> DispatchResult {
 			T::AdminOrigin::ensure_origin(origin.clone())?;
 
 			ensure!(
-				ConnectorsAllowlist::<T>::contains_key(connector.domain(), connector.clone()),
-				Error::<T>::ConnectorNotFound,
+				Allowlist::<T>::contains_key(instance.domain(), instance.clone()),
+				Error::<T>::UnknownInstance,
```

Comment:
Thanks for merging both errors. Totally makes sense as both were thrown when the instance was not part of the allowlist.